DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests.cs

```diff
@@ -32,6 +32,10 @@ public void Dispose()
         [InlineData("/Login")]
         [InlineData("/ForgotPassword")]
         [InlineData("/Register")]
+        [InlineData("/Register/LearnerInformation")]
+        [InlineData("/Register/Password")]
+        [InlineData("/Register/Summary")]
+        [InlineData("/Register/Confirmation")]

```

Comment:
Does this work without TempData? (it feels like this would just redirect the user)

I think we should simulate the registration process and then scan each page (a different test). I'm happy to have a call about this, but also, feel free to scan over the HEEDLS-NONE-auth-accessibility branch first commit to see how the login form is handled. Also if you comment out the chromeOptions.AddArgument("--headless");  line, you should be able to visualise what's happening in the browser when you run your tests with breakpoints.